src/components/MenuBar/MenuBar.style.ts

```diff
@@ -0,0 +1,117 @@
+import styled from 'styled-components';
+
+// Menu Bar container styling
+export const StyledMenuBar = styled.div`
```

Comment:
Non-blocking thought: Maybe this could be a HTML `nav` element rather than a `div`?